library/CMakeLists.txt

```diff
@@ -390,4 +390,4 @@ if(DPP_INSTALL)
 
 	# CPack initialization for distribution
 	include(CPack)
-endif()
+endif()
```

Comment:
Why was this modified? By UNIX convention, a text file must have a newline at the end to be recognized as a text file. Please add the newline back to the file.